apps/server/src/modules/board/board-context-api-helper.module.ts

```diff
@@ -0,0 +1,13 @@
+import { Module } from '@nestjs/common';
```

Comment:
I would put this module into a seperate module folder, to avoid cycles through the folder structure/imports

Reply:
moved